staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -211,6 +211,9 @@ func validateBackendBatchConfig(pluginName string, config pluginbuffered.BatchCo
 	if config.ThrottleBurst <= 0 {
 		return fmt.Errorf("invalid audit batch %s throttle burst %v, must be a positive number", pluginName, config.ThrottleBurst)
 	}
+	if config.MaxGoRoutines < config.ThrottleBurst {
```

Comment:
Those are independent values, it's wrong to compare them. Also throttling can be disabled altogether. I think just comparing it with 0 is better

Reply:
SGTM